go.mod

```diff
@@ -47,9 +47,10 @@ require (
 	github.com/uber/jaeger-client-go v2.30.0+incompatible
 	github.com/uber/jaeger-lib v2.4.1+incompatible
 	github.com/xdg-go/scram v1.1.1
-	go.opentelemetry.io/collector v0.63.0
-	go.opentelemetry.io/collector/pdata v0.63.0
-	go.opentelemetry.io/collector/semconv v0.63.0
+	go.opentelemetry.io/collector v0.64.0
+	go.opentelemetry.io/collector/pdata v0.64.0
+	go.opentelemetry.io/collector/receiver/otlpreceiver v0.64.0
```

Comment:
This is new. Usually we also need to upgrade the contrib piece in L34:
```
github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger v0.63.0
```

Reply:
Yes the otlp receiver is now its own module as of v0.64.0

Reply:
I updated the translator package as well 👍🏻